controller/index.php

```diff
@@ -399,11 +399,18 @@ protected function list_contributions($categories, $sort_url)
 		$sort->set_defaults(24);
 		$branch = (int) str_replace('.', '', $this->branch);
 
+		// Get the featured contributions
+		\contribs_overlord::featured_contribs();
+
 		$data = \contribs_overlord::display_contribs($mode, $categories, $branch, $sort, 'contribs', $this->status);
 
 		// Canonical URL
 		$data['sort']->set_url($sort_url);
-		$this->template->assign_var('U_CANONICAL', $data['sort']->build_canonical());
+
+		$this->template->assign_vars([
+			'U_CANONICAL' => $data['sort']->build_canonical(),
+			'U_IS_TITANIA_INDEX' => (!$sort_url) ? true : false,
```

Comment:
Please just make this: `'S_IS_TITANIA_INDEX' => empty($sort_url),` or `'S_IS_TITANIA_INDEX' => $sort_url === 0,`

It's a boolaean switch so we should use the `S_` prefix. Also, it's better to just have a boolean condition rather than the whole if it strue or false then make it true or false conditional statement.

Reply:
Done